exasol/analytics/schema/column_types.py

```diff
@@ -0,0 +1,119 @@
+import re
+from dataclasses import dataclass
+from enum import (
+    Enum,
+    auto,
+)
+from typing import (
+    Any,
+    Callable,
+    Optional,
+)
+
+
+class CharSet(Enum):
+    UTF8 = auto()
+    ASCII = auto()
+
+    @classmethod
+    def from_string(cls, name: str) -> "CharSet":
```

Comment:
Is `CharSet(name)` not good?

Reply:
Yes, you are right.
Thanks!
See next push, fixing this.